src/components/biggive-article-card/biggive-article-card.tsx

```diff
@@ -52,6 +52,10 @@ export class BiggiveArticleCard {
 
   @Prop() clipTopRightCorner: boolean = true;
 
+  private isEmpty(value: string): boolean {
+    return value === undefined || value === null || value === '';
```

Comment:
this seems like a type contradiction. Do you know if it's only in WP that it's possible for the value to be undefined or null when it's typed as `string`? I guess our static checks in Angular should prevent that.

Anyway as long as it is possible maybe worth replacing the string type with `string|undefined|null`? 

Could also implement this as `return !!value;`.

Reply:
It seems like all contexts had the problem at runtime where empty image tags were rendered: Stencil tests, plain components in Storybook, WordPress and Angular. In case of Angular maybe it's because nothing was passed in at all so the `undefined` default from this file is maintained?

In any case, yes makes sense to fix `value`'s type, done in e22c183a1e696da60f2cafdaf0d24a35bd55662c

Reply:
Right, I can't remember exactly where we got to with ability to make properties as required in stencil and have that enforced in Angular compilation. I guess the other option might be to set a default of '' so that it's always a string. But this looks good for now.